TFT/src/User/Menu/FeatureSettings.c

```diff
@@ -47,7 +47,7 @@ typedef enum
   SKEY_START_GCODE_ENABLED,
   SKEY_END_GCODE_ENABLED,
   SKEY_CANCEL_GCODE_ENABLED,
-  SKEY_RESET_SETTINGS,        // Keep reset always at the bottom of the settings menu list.
```

Comment:
after looking at this PR, I find it hard to understand the logic for when to make the first letter of the sentence Capital or small. 🤣

Reply:
personally, I always prefer to provide inline comments in lower case and without "." at the end of a sentence. I Use first capital letter and "." when the there are more sentences (e.g. "this is first sentence. This is a second sentence"). I also try to use a common style in a whole file. So if the majorithy of comments were lower case, I use that, Most of the time the comments are lower case, so you see that more often